scripts/gen_keys.js

```diff
@@ -38,6 +38,13 @@ function main(cb) {
   var privKey = JwTool.JWK.fromPEM(generateRSAKeypair().private, {
     kid: '2015.12.16-1'
   });
+
+  try {
+    fs.mkdirSync('./config');
+  } catch (accessEx) {
+
```

Comment:
I added this because when we're building the Docker image, the config directory doesn't exist yet. So all this does is try to create it. If it does exist then it fails and it's still all good.

I can add a message, but I don't think it'll add anything necessary here.